MacroExamplesPlugin/PrintArgumentsMacro.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  PrintArgumentsMacro.swift
+//  MacroExamplesPlugin
+//
+//  Created by Stephen Kockentiedt on 18.12.22.
+//
+
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import _SwiftSyntaxMacros
+
+public struct PrintArgumentsMacro: ExpressionMacro {
+    public static func expansion(
+        of node: MacroExpansionExprSyntax, in context: inout MacroExpansionContext
+    ) -> ExprSyntax {
+        var syntax = node.as(Syntax.self)
+        while syntax != nil && syntax!.as(FunctionDeclSyntax.self) == nil {
+            syntax = syntax!.parent
```

Comment:
Now that macro implementations are getting "detached" nodes, the parent of the main syntax node will always be `nil`, so `#printArguments` will always return the empty string.

Reply:
That's a little unfortunate. So the solution of providing a 'spine' of the hierarchy is not the way to go?